inst/csv/replacementPatterns.csv

```diff
@@ -247,4 +247,50 @@ From,To,Pattern,Replacement
 "sql server","impala",".dbo.","."
 "sql server","impala","##","<hash><hash>"
 "sql server","impala","#","%temp_prefix%%session_id%"
-"sql server","impala","<hash><hash>","##"
\ No newline at end of file
+"sql server","impala","<hash><hash>","##"

```

Comment:
@chrisknoll it looks like this was copied from the Oracle translation rules, where it was necessary to distinguish schema names with compulsory double hashes from temp tables in our SQL: https://github.com/OHDSI/SqlRender/commit/0f52ad6923719fb6ddb34aaf4712f852d2f9b6f6

@gowthamrao , in line 249 you are using the workaround we implemented for Oracle which doesn't really support temp tables. This workaround emulates temp tables by creating non-temp tables with system-generated names in a user-defined schema. It would be preferred to use real temp tables if possible. As far as I can see Netezza supports temp tables just fine?